paddle/majel/allocation.cc

```diff
@@ -0,0 +1,97 @@
+#include <boost/variant.hpp>
+
+#include "allocation.h"
+#include "hl_cuda.h"
+#include "paddle/utils/Logging.h"
```

Comment:
I think we made an agreement yesterday that Majel shouldn't depend on Paddle.  For logging, we should just use google log.